sdk/tables/azure-data-tables/azure/data/tables/aio/_policies_async.py

```diff
@@ -161,7 +204,7 @@ def __init__(self, initial_backoff=15, increment_base=3, retry_total=3,
         super(ExponentialRetry, self).__init__(
             retry_total=retry_total, retry_to_secondary=retry_to_secondary, **kwargs)
 
-    def get_backoff_time(self, settings, **kwargs):
+    def get_backoff_time(self, settings, **kwargs): # pylint: disable=unused-argument
```

Comment:
if kwargs is unused, you can remove it like you did for the sync `get_backoff_time` and other methods